src/elements/checkbox-element.js

```diff
@@ -0,0 +1,7 @@
+import BooleanElement from './boolean-element';
+
+export default class CheckBoxElement extends BooleanElement {
+  get isLengthValidationSupported() {
```

Comment:
This appears to be the same as the Element default definition, does it need to be overridden here?

Reply:
I am still trying to figure out what this does.

Reply:
This activates the ability to define "min" and "max" validations, which on numbers constrain the value and array-ish things constrain the number of photos/repeatables/etc.